### ISSUE TITLE
How to set it up for latex fragments in a org file

### ISSUE THREAD

**Issue by student-sx** [polymode/poly-org#27]
Can I use poly-org to automatically switch to latex mode if the point is between [latex delimiters](https://orgmode.org/manual/LaTeX-fragments.html)? What to add to my init file to make it work?

**Comment by vspinu** [comment]
I have created [inline-latex](https://github.com/polymode/poly-org/commit/d4d00997b82de7eeaa456d960c917e5e155374c0) branch. Unfortunately I see font lock errors and indentation is not working as expected, so it's not veryuseful I would need to spend more time with it and cannot promise that it will happen soon unfortunately.

**Comment by student-sx** [comment]
Thanks. Something like `\begin{equation} ... \end{equation}` seems to be recognized. However `\[ ... \]` or `\(... \)` doesn't work for me, as well as `$a^2$` or `$$ a^2 + b^2 = c^2 $$`. Compare the regexp used in [poly-markdown.el](https://github.com/polymode/poly-markdown/blob/master/poly-markdown.el).

**Comment by vspinu** [comment]
yes, it does not because even the `\begin .. \end` has issues, so I didn't proceed with others.

**Comment by QMeqGR** [comment]
Hello, I just discovered polymode and poly-org and would like to use it for inline LaTeX as well. If the trouble is with the symmetric delimiters $..$ and $$..$$, I could live without them. I tried to modify the regex for \begin..\end to work with \[..\]  but I don't know regular expressions well enough to make it work. Is it as straightforward as modifying the regex to search for \[..\] and \(..\) ?
